src/components/DatePicker/DatePicker.tsx

```diff
@@ -42,12 +38,14 @@ export interface DatePickerProps {
   disableDatesAfter?: Date;
   /** The selection can span multiple months */
   multiMonth?: boolean;
-  /** First day of week. Sunday by default */
-  weekStartsOn?: Weekday;
+  /** First day of week, from 0 to 6. 0 is Sunday, 1 is Monday ... 6 is Saturday
+   * @default 0
+   */
```

Comment:
Not sure it matters or if typedoc will like this, but multiline comment blocks typically are typically formatted like this:

```suggestion
  /**
   * First day of week, from 0 to 6. 0 is Sunday, 1 is Monday ... 6 is Saturday
   * @default 0
   */
```